chia/full_node/coin_store.py

```diff
@@ -103,6 +109,7 @@ async def new_block(self, block: FullBlock, tx_additions: List[Coin], tx_removal
 
         # Sanity check, already checked in block_body_validation
         assert sum([a.amount for a in tx_additions]) <= total_amount_spent
+        log.info(f"new_block() took {time() - start:0.6}s additions: {len(tx_additions)} removals: {len(tx_removals)}")
```

Comment:
Maybe this should be log.debug instead of log.info. We want to keep info as minimal as possible, only things that a casual observer might want in order to verify correct blockchain operation

Reply:
makes sense. I was also considering only logging in case the time exceeds some limit (say, 10 seconds or so). Do you think it would make sense to turn it into a `WARNING` if it takes too long, and `DEBUG` otherwise?

Reply:
I'm a big fan of having "too long" times showing up in error vs everything being in debug.

Reply:
although, we do print mempool_manager's `add_spendbundle` timing as `INFO`. That might have been me too though.

Reply:
I'll update to *only* log as `WARNING` when it takes more than 10 seconds, and make the message helpful